File-Formats/DocIO/Convert-Word-Document-to-Image-in-Blazor.md

```diff
@@ -0,0 +1,358 @@
+![Install SkiaSharp.NativeAssets.WebAssembly NuGet Package](Blazor_Images/SkiaSharp-Nuget-WordtoPDF.png)
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering Syncfusion license key in your application to use our components.
+
+Step 4: Add the following ItemGroup tag in the **Blazor WASM csproj** file.
+{% tabs %}
+
+{% highlight XAML  %}
+<ItemGroup>
+    <NativeFileReference Include="$(SkiaSharpStaticLibraryPath)\2.0.23\*.a" />
+</ItemGroup>
+{% endhighlight %}
+
+{% endtabs %}
+
+N> Install this wasm-tools and wasm-tools-net6 by using the "dotnet workload install wasm-tools" and "dotnet workload install wasm-tools-net6" commands in your command prompt respectively if you are facing issues related to Skiasharp during runtime.
+
+Step 5: Enable the following property in the Blazor WASM csproj file.
+
+{% tabs %}
+{% highlight XAML %}
+<PropertyGroup>
+    <WasmNativeStrip>true</WasmNativeStrip>
+</PropertyGroup>
+{% endhighlight %}
+{% endtabs %}
+
+Step 6: Create a razor file with name as ``DocIO`` under ``Pages`` folder and add the following namespaces in the file.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+@page "/docio"
+@using Syncfusion.DocIO
+@using Syncfusion.DocIORenderer
+@using Syncfusion.DocIO.DLS
+@inject Microsoft.JSInterop.IJSRuntime JS
+@inject HttpClient client
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 7: Add the following code to create a new button.
+
+{% tabs %}
+
+{% highlight CSHTML %}
+
+<h2>Syncfusion DocIO library (Essential DocIO)</h2>
+<p>Syncfusion Blazor DocIO library (Essential DocIO) used to create, read, edit, and convert DocIO files in your applications without Microsoft Office dependencies.</p>
+<button class="btn btn-primary" @onclick="@WordToImage">Convert Word to Image</button>
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 8: Create a new async method with name as ``WordToImage`` and include the following code snippet to **convert a Word document to Image in Blazor** WASM app.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+//Open the file as Stream
+using (FileStream sourceStreamPath = new FileStream(@"wwwroot/Template.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+{
+    //Open an existing Word document.
+    using (WordDocument document = new WordDocument(sourceStreamPath, FormatType.Docx))
+    {
+        //Instantiation of DocIORenderer for Word to Image conversion
+        using (DocIORenderer render = new DocIORenderer())
+        {
+            Stream imageStream = document.RenderAsImages(0, ExportImageFormat.Jpeg);
+            //Reset the stream position.
+            imageStream.Position = 0;
+            return (MemoryStream)imageStream;
```

Comment:
MemoryStream imageStream = (MemoryStream)document.RenderAsImages(0, ExportImageFormat.Jpeg);
//Reset the stream position.
imageStream.Position = 0;
//Download PDF file in the browser.
await JS.SaveAs("WordToImage.Jpeg", imageStream.ToArray());

Reply:
updated